packages/contracts/contracts/CdpManagerStorage.sol

```diff
@@ -516,7 +516,6 @@ contract CdpManagerStorage is EbtcBase, ReentrancyGuard, ICdpManagerData, AuthNo
                 uint256 _perUnitError
             ) = _calcSyncedGlobalAccounting(_newIndex, _oldIndex);
             _takeSplitAndUpdateFeePerUnit(_feeTaken, _newFeePerUnit, _perUnitError);
-            _updateSystemSnapshotsExcludeCollRemainder(0);
```

Comment:
This change looks like not the solution and should be reverted https://github.com/code-423n4/2023-10-badger-findings/issues/155#issuecomment-1831082736